api/server/server.go

```diff
@@ -196,13 +196,14 @@ func postContainersKill(eng *engine.Engine, version version.Version, w http.Resp
 	if err := parseForm(r); err != nil {
 		return err
 	}
-	job := eng.Job("kill", vars["name"])
-	if sig := r.Form.Get("signal"); sig != "" {
-		job.Args = append(job.Args, sig)
-	}
-	if err := job.Run(); err != nil {
+
+	name := vars["name"]
+	sig := vars["signal"]
```

Comment:
I thought about that, but I wasn't sure if we wanted callers to ContainerKill be able to pass in both ints and the string version w/o forcing everyone to convert it themselves.